ansible/group_vars/development

```diff
@@ -20,12 +20,12 @@ wordpress_sites:
       enabled: false
     system_cron: true
     env:
-      wp_home: http://roots-example-project.dev
-      wp_siteurl: http://roots-example-project.dev/wp
+      wp_home: http://example.dev
+      wp_siteurl: http://example.dev/wp
       wp_env: development
-      db_name: roots_example_project_dev
-      db_user: roots_example_project_dbuser
-      db_password: roots_example_project_dbpassword
+      db_name: example_dev
+      db_user: example_dbuser
+      db_password: example_dbpassword
```

Comment:
All of the existing settings need to remain here. I think the only actual change was "project" to "site" on L9.